tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentMethodEntry.h

```diff
@@ -160,13 +160,15 @@ class CInstrumentMethodEntry
 
     HRESULT GetModuleName(BSTR* pbstrModuleName)
     {
-        m_bstrModuleName.CopyTo(pbstrModuleName);
+        HRESULT hr;
```

Comment:
Should these be:
```suggestion
        HRESULT hr = S_OK;
```

Reply:
Not necessary. The hresult will be set immediately on the next line.